Application/EdFi.Ods.Api/Security/Authorization/AuthorizationSegmentSqlProviderBase.cs

```diff
@@ -82,6 +82,27 @@ public AuthorizationSegmentSqlProviderBase(IAuthorizationViewsProvider authoriza
                     // Perform defensive checks against the remote possibility of SQL injection attack
                     ValidateTableNameParts(claimEndpointName, subjectEndpointName, authorizationSegment.AuthorizationPathModifier);
 
+                    var tupleBackedByNewAuthViews = new List<string>
```

Comment:
A small improvement would be to use HashSet<string> to have constant search times (instead of linear), the difference would be small though since the list has few items